test/odbc/psqlodbc/psqlodbc_tests_common.h

```diff
@@ -412,6 +412,10 @@ void testComparisonFunctions(ServerType serverType, const string &tableName, int
  * @param expectedResults 2D vector containing the expected results for each operation.
  * @param expectedLen A vector containing the expected length of all results from each operation. 
 */
+
+void testStringFunctions(ServerType serverType, const string &tableName, const vector<string> &operationsQuery, 
+vector<vector<string>> &expectedResults, const vector<string> &pk, const string &Col_NAME);
+
```

Comment:
You're seperating the docstrings comments for testArithmeticOperators. Please put this seperately (should be before the templates) and also add your own docs.